src/stylesheets/elements/_buttons.scss

```diff
@@ -45,12 +44,6 @@ button,
     text-decoration: none;
   }
 
-  &:focus,
-  &.usa-button-focus {
```

Comment:
What if we just did a `usa-focus` class that could work for focus state on any element? Or should there but a `button-focus` class specifically?